components/project-card/project-card.scss

```diff
@@ -51,6 +51,11 @@
     margin: 0.5rem 0;
   }
 
+  p {
+    font-size: 0.9375rem;
```

Comment:
This feels like an arbitrary pixel difference... @kristinashu does this *need* to be 1px below the baseline browser default?